web-api/src/business/useCases/processStreamRecords/processMessageEntries.ts

```diff
@@ -1,20 +1,16 @@
 import { RawMessage } from '@shared/business/entities/Message';
+import { getLogger } from 'aws-xray-sdk';
```

Comment:
I don't think this is the right logger!

Reply:
100%, will update as soon as I have a chance

Reply:
Done!